### ISSUE TITLE
2 instances of mod_wrapper results in duplicate ID

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#17425]
### Steps to reproduce the issue

If you have 2 instances of `mod_wrapper` on 1 page, then there will be 2 iframes with the same ID ('blockrandom').

- Create 2 wrapper modules
- Assign them both to the same page
- Inspect the element and they will have the same ID

When a PR is submitted, they'll also have to update: https://github.com/joomla/joomla-cms/blob/staging/media/com_wrapper/js/iframe-height.js#L4

### System information

- Joomla staging

**Comment by ghost** [comment]
Issue confirmed.

**Comment by DrDreave** [comment]
When examining the issue I noticed the `media/com_wrapper/js/iframe-height.js` isn't working properly in today's browsers. Enabling the auto height option in recent versions of Chrome and Firefox on MacOS has no effect.

e.g. Chrome throws:
```
Uncaught DOMException: Failed to read the 'contentDocument' property from 'HTMLIFrameElement': Blocked a frame with origin "http://joomla-git-sample.local" from accessing a cross-origin frame.
    at HTMLIFrameElement.contentDocumentDesc.get [as contentDocument] (<anonymous>:66:14)
    at iFrameHeight (http://joomla-git-sample.local/media/com_wrapper/js/iframe-height.min.js?84614c47e342ac58508eacfcd3d2d1ec:1:99)
    at HTMLIFrameElement.onload (http://joomla-git-sample.local/wrapper-module:162:19)
```

I think alternative approaches of setting the height should be discussed before reworking the id assignment to the wrappers.

**Comment by brianteeman** [comment]
Closed see #19136
